java.base/src/main/java/sun/nio/fs/UnixNativeDispatcher$_native.java

```diff
@@ -62,6 +62,9 @@
 @Tracking("src/java.base/unix/native/libnio/fs/UnixNativeDispatcher.c")
 class UnixNativeDispatcher$_native {
     static byte[] getcwd() throws UnixException {
+        if (Build.Target.isWasm()) {
+            throw new UnixException(-1);
```

Comment:
I don't know what -1 would do here but probably nothing good! 😂

Maybe instead return e.g. `"/qbicc/build".getBytes(StandardCharsets.UTF_8)` for now?